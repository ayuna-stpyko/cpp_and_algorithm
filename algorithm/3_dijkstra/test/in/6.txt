8 8
3 1 4
3 2 3
1 2 1
0 4 2
4 5 3
5 6 1
6 7 5
5 7 2
