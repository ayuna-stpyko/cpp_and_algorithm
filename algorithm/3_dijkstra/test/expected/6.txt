0
INF
INF
INF
2
5
6
7
